src/pages/common/_scanner_page.c

```diff
@@ -99,18 +42,27 @@ static void press_enable_cb(guiObject_t *obj, const void *data)
     GUI_Redraw(obj);
 }
 
-static void press_mode_cb(guiObject_t *obj, const void *data)
+static const char *attenuator_cb(guiObject_t *obj, int dir, void *data)
 {
+    (void)obj;
     (void)data;
-    sp->scan_mode ^= 1;
-    GUI_Redraw(obj);
+    sp->attenuator = GUI_TextSelectHelper(sp->attenuator, 0, 2, dir, 1, 1, NULL);
```

Comment:
can we use PROTO_OPTS to let the user choose the options? So that we don't need to create another set of UI elements. Also, this gives different options for different scanners.

Reply:
Proto opts would require leaving scanner page, but those I put in options need to be changed quickly and there effect immediately seen.